app/src/main/java/com/keylesspalace/tusky/adapter/NotificationsAdapter.java

```diff
@@ -136,6 +136,7 @@ public void onBindViewHolder(@NonNull RecyclerView.ViewHolder viewHolder, int po
                     (NotificationViewData.Concrete) notification;
             Notification.Type type = concreteNotificaton.getType();
             switch (type) {
+                default:
```

Comment:
Huh? I don't get it, default is a mention?

Reply:
When type is null, switch statement returns NullPointerException.
This seems to be a Java's specification.

Reply:
Indeed it is and indeed it does throw. I just don't understand how "default" changes that:
https://ideone.com/I82Z5B